modules/extension/xsd/xsd-fes/src/main/java/org/geotools/filter/v2_0/bindings/PropertyIsLikeTypeBinding.java

```diff
@@ -55,13 +64,20 @@ public PropertyIsLikeTypeBinding(FilterFactory factory) {
         this.factory = factory;
     }
 
+    @Override
     public QName getTarget() {
         return FES.PropertyIsLikeType;
     }
 
+    @Override
     public Object getProperty(Object object, QName name) throws Exception {
         PropertyIsLike isLike = (PropertyIsLike) object;
 
+        if (FES.expression.equals(name) && isLike instanceof LikeFilterFesImpl) {
+            LikeFilterFesImpl isLikeFes = (LikeFilterFesImpl) isLike;
+            return new Object[] {isLikeFes.getExpression1(), isLikeFes.getExpression2()};
```

Comment:
This appeared to do the job, but if we need to implement `encode(...)` then so be it.